include/nlohmann/detail/json_custom_base_class.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <type_traits> // conditional, is_same
+
+NLOHMANN_JSON_NAMESPACE_BEGIN
```

Comment:
The CI complains that this symbol cannot be found.

Please include `<nlohmann/detail/abi_macros.hpp>`.

Reply:
Added